src/Confluent.Kafka/Consumer.cs

```diff
@@ -151,21 +151,14 @@ public bool Consume(out Message<TKey, TValue> message, int millisecondsTimeout)
         public bool Consume(out Message<TKey, TValue> message, TimeSpan timeout)
             => Consume(out message, timeout.TotalMillisecondsAsInt());
 
-        /// <summary>
-        ///     Refer to <see cref="Consume(out Message{TKey, TValue}, int)" />.
-        ///     
-        ///     [UNSTABLE-API] - prefer to use <see cref="Poll()"/> / <see cref="OnMessage"/> instead of this method.
-        /// </summary>
-        public bool Consume(out Message<TKey, TValue> message)
-            => Consume(out message, -1);
 
         /// <summary>
         ///     Poll for new consumer events, including new messages
         ///     ready to be consumed (which will trigger the OnMessage
         ///     event).
         /// </summary>
         /// <param name="millisecondsTimeout"> 
-        ///     The maximum time to block (in milliseconds).
+        ///     The maximum time to block (in milliseconds), or -1 to block indefinitely.
```

Comment:
Add that -1 should not typically be used. Same remarks for the Poll(TimeSpan) : TimeSpan.Infinite should typically not be used